### DISCUSSION TITLE
Wrong update server

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#833] [Q&A (Community Support)]
Hello in Panopticon I received a notification to update Akeeba Backup from version 9.9.6 to 9.9.7. However, the log file contains a message advising to check the update server. Here is an example.
_Extension updates for site #35 (****): failed installing update for package “Akeeba Backup for Joomla! package”. Joomla! reported an error: The checksum verification failed. Please make sure you are using the correct update server!_

Inside Update sites URL is 
_https://cdn.akeeba.com/updates/pkgakeebabackuppro.xml_

**Comment by nikosdion** [comment]
Wait for Joomla's update cache to pick up the fact there's version 9.9.9.
